jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -75,5 +65,30 @@ This content is unavailable. Enable Early Access to see it.</description>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
 
+       <!-- JBoss Fuse Development -->
```

Comment:
I can remove it and we can get https://issues.jboss.org/browse/JBIDE-20664 fixed this week, so there's a better way to include IS jars in the main JBT/JBDS Disco CI sites. But there's no *released* JBDS IS discovery content yet, so it's a bit tricky. :D